datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/DatalakeDatabaseBackupFlowConfig.java

```diff
@@ -55,7 +67,7 @@
                     .from(DATALAKE_DATABASE_BACKUP_IN_PROGRESS_STATE)
                     .to(DATALAKE_FULL_BACKUP_IN_PROGRESS_STATE)
                     .event(DATALAKE_FULL_BACKUP_IN_PROGRESS_EVENT)
-                    .failureState(DATALAKE_DATABASE_BACKUP_FAILED_STATE)
+                    .failureState(DATALAKE_FULL_BACKUP_IN_PROGRESS_STATE)
```

Comment:
if we do this then  .from(DATALAKE_DATABASE_BACKUP_FAILED_STATE) .to(DATALAKE_FULL_BACKUP_IN_PROGRESS_STATE)... will never run, do we want this? If yes then we should remove that "from-to" (line 83-87)

Reply:
You are right.

Reply:
you are right. Wil fix it.

Reply:
addressed.